docs/concepts/sidecarSet/README.md

```diff
@@ -53,7 +53,7 @@ spec:
       app: nginx
   containers:
   - name: sidecar1
-    image: centos:7
+    image: centos:6.7
```

Comment:
why is this changed?

Reply:
OCD. :)
in my README.md, it makes me feel better to change from centos:6.7 to centos:6.8 instead of changing from centos:7 to centos:6.8